application/modules/front/views/collection/hadith_reference.php

```diff
@@ -149,9 +149,8 @@
 				//echo "<a href=\"javascript:permalink('$permalink');\">Permalink</a>";
 				//echo "<a href=\"$permalink\">Permalink</a>";
 			if (!isset($hideReportError) or !$hideReportError) echo "<span class=reportlink href=\"javascript: void(0);\" onclick=\"reportHadith(".$urn.", '".$divName."')\">Report Error</span> | ";
-			if (!$hideShare) echo  "<span class=sharelink onclick=\"share('$permalink')\">Share</span> | "; 
-				echo "<span class=copylink title=\"Copy hadith to clipboard\">Copy</span> "; 
-				echo "<span class=copycbcaret title=\"Change copy options\">▼</span>".
-			"</div>". 
-		"</div>";
+			if (!$hideShare) echo "<span class=sharelink onclick=\"share('$permalink')\">Share</span> | "; 
+			if (!$hideShare) echo "<span class=copylink title=\"Copy hadith to clipboard\">Copy</span> "; 
+			if (!$hideShare) echo "<span class=copycbcaret title=\"Change copy options\">▼</span>";
```

Comment:
Can these three echo statements be put in the same `if` block?

Reply:
Done.